### DISCUSSION TITLE
NOTICE: upcoming requirement of npm v8

### DISCUSSION THREAD

**Discussion by boneskull** [appium/appium#17300] [General]
In the next published version of Appium 2.0.0 beta, the minimum supported version of `npm` will be v8.

To check your version of `npm`, run:

```bash
npm --version
```

If this is lower than v8.x, then tell `npm` to upgrade itself with:

```
npm install --global npm
```

The minimum supported version of Node.js is still v14.

**Comment by Auto81** [comment]
What beta version number is this expected on specifically? 40, 41?

**Reply by KazuCocoa** [reply]
since beta 42. Appium will show an error message if the running environment does not have over npm v8.
